layersvt/device_simulation_examples/sdk_sample_configs/minimum_vulkan_1_2.json

```diff
@@ -0,0 +1,1133 @@
+        "shaderStorageImageArrayNonUniformIndexing": 0,
+        "shaderInputAttachmentArrayNonUniformIndexing": 0,
+        "shaderUniformTexelBufferArrayNonUniformIndexing": 0,
+        "shaderStorageTexelBufferArrayNonUniformIndexing": 0,
+        "descriptorBindingUniformBufferUpdateAfterBind": 0,
+        "descriptorBindingSampledImageUpdateAfterBind": 0,
+        "descriptorBindingStorageImageUpdateAfterBind": 0,
+        "descriptorBindingStorageBufferUpdateAfterBind": 0,
+        "descriptorBindingUniformTexelBufferUpdateAfterBind": 0,
+        "descriptorBindingStorageTexelBufferUpdateAfterBind": 0,
+        "descriptorBindingUpdateUnusedWhilePending": 0,
+        "descriptorBindingPartiallyBound": 0,
+        "descriptorBindingVariableDescriptorCount": 0,
+        "runtimeDescriptorArray": 0
+    },
+    "VkPhysicalDeviceHostQueryResetFeatures": {
+        "hostQueryReset": 0
+    },
+    "VkPhysicalDeviceImagelessFramebufferFeatures": {
+        "imagelessFramebuffer": 0
+    },
+    "VkPhysicalDeviceMultiviewFeatures": {
+        "multiview": 1,
+        "multiviewGeometryShader": 0,
+        "multiviewTessellationShader": 0
+    },
+    "VkPhysicalDeviceProtectedMemoryFeatures": {
+        "protectedMemory": 0
+    },
+    "VkPhysicalDeviceSamplerYcbcrConversionFeatures": {
+        "samplerYcbcrConversion": 0
+    },
+    "VkPhysicalDeviceScalarBlockLayoutFeatures": {
+        "scalarBlockLayout": 0
+    },
+    "VkPhysicalDeviceSeparateDepthStencilLayoutsFeatures": {
+        "separateDepthStencilLayouts": 0
+    },
+    "VkPhysicalDeviceShaderAtomicInt64Features": {
+        "shaderBufferInt64Atomics": 1,
+        "shaderSharedInt64Atomics": 1
+    },
+    "VkPhysicalDeviceShaderDrawParametersFeatures": {
+        "shaderDrawParameters": 0
+    },
+    "VkPhysicalDeviceShaderFloat16Int8Features": {
+        "shaderFloat16": 0,
+        "shaderInt8": 0
+    },
+    "VkPhysicalDeviceShaderSubgroupExtendedTypesFeatures": {
+        "shaderSubgroupExtendedTypes": 0
+    },
+    "VkPhysicalDeviceTimelineSemaphoreFeatures": {
+        "timelineSemaphore": 0
+    },
+    "VkPhysicalDeviceUniformBufferStandardLayoutFeatures": {
+        "uniformBufferStandardLayout": 1
+    },
+    "VkPhysicalDeviceVariablePointersFeatures": {
+        "variablePointersStorageBuffer": 0,
+        "variablePointers": 0
+    },
+    "VkPhysicalDeviceVulkanMemoryModelFeatures": {
+        "vulkanMemoryModel": 1,
+        "vulkanMemoryModelDeviceScope": 0,
+        "vulkanMemoryModelAvailabilityVisibilityChains": 0
+    },
+    "Vulkan12Features": {
+        "samplerMirrorClampToEdge": 0,
+        "shaderOutputViewportIndex": 0,
+        "shaderOutputLayer": 0,
+        "subgroupBroadcastDynamicId": 1,
+        "drawIndirectCount": 0,
+        "descriptorIndexing": 0,
+        "samplerFilterMinmax": 0
+    },
+    "Vulkan12Properties": {
+        "framebufferIntegerColorSampleCounts": 1
+    },
+    "ArrayOfVkFormatProperties": [
```

Comment:
This part is not really readable by a Vulkan developer.

I think we need both the formatID and a label (the enum string) so that we could immediately see what are these formats.

I am guessing same for the tiling flags.

Reply:
That is something that can be discussed about improving devsim config file readability in the future, but that's not what this PR is about.

This PR is about creating a Vulkan 1.2 minimum devsim config file, and it does just that with the current devsim config file format as defined by the schemes.